scripts/regression-list.sh

```diff
@@ -36,16 +36,18 @@ else
       23 31 31
       24 17 32
       24 20 32
-      #23 17 33
-      #23 20 33
-      #3  11 32
-      #17 13 16
-      #17 13 17
+      23 17 33
+      23 20 33
+      3  11 32
+      17 13 16
+      17 13 17
```

Comment:
Can you check and add the case reported by Fenzi? @yvantor knows which one...

Reply:
48x48x48